src/main/scala/tilelink/SRAM.scala

```diff
@@ -105,7 +106,8 @@ class TLRAM(
     val d_correctable   = d_decoded.map(_.correctable)
     val d_uncorrectable = d_decoded.map(_.uncorrectable)
     val d_need_fix      = d_correctable.reduce(_ || _)
-    val d_error         = d_uncorrectable.reduce(_ || _)
+    val d_lane_error    = Cat(d_uncorrectable.reverse) & d_lanes
+    val d_error         = d_lane_error.orR
 
     notifyNode.foreach { nnode =>
       nnode.bundle.correctable.foreach { c =>
```

Comment:
Here a request could notify a correctable ECC error using the request's address, even when the request's address+size does not include the error.

Reply:
That is intentional. Correctable ECC errors should always be corrected as soon as they are detected.

Reply:
I think it's important to also note that corrected errors are NOT reported to the user.

Reply:
Oh. I see that there is now an auxiliary 'notifyNode'. @hcook I'm inclined to leave these out-of-band notifications to repair errors as early as detected. For example, I might have had an idle check-and-scrub loop running also in the absence of requests. Do you agree?

Reply:
If an error was discovered and corrected, a notification should occur. I don't see it as having anything to do semantically with a particular request, as the notification doesn't contain nor record information like a TL source id or a size. So I agree that notifications should occur for scrubbing activity. Does the address field need to be somehow more precise?